### DISCUSSION TITLE
Trigger command when Enter pressed twice consecutively

### DISCUSSION THREAD

**Discussion by jamilabreu** [ueberdosis/tiptap#1713] [General Questions]
Is there a recommended way to execute a command when Enter (or any two keys) is double tapped?

**Comment by smplyjr** [comment]
I am currently using tiptap for a chat widget message area and I want to catch the "enter" key event. I want to submit the chat when a user hit "enter". I found the onUpdate event listener but I cannot locate the key that is pressed.

**Reply by smplyjr** [reply]
For future reference of other developers, I use VueJS native keydown event to detect which key is pressed.

```javascript
<editor-content class="editor" :editor="editor" @keydown.native="onKeydown" />
```
```javascript
onKeydown(e) {
  if (!e.shiftKey && e.which == 13) {
    this.editor.commands.undo(); // i just "undo" the enter event to remove the space it made
    this.$emit("onEnter");
  }
}
```

Reference: https://www.tiptap.dev/installation/vue2#5-use-v-model-optional
